### PR TITLE
Banana dataset is now a classification problem

### PR DESCRIPTION
**Pull request by alexzwanenburg**
The banana dataset was previously listed as a regression problem #162.

This has now been fixed.

### CODE REVIEW

File: datasets/banana/metadata.yaml [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-    type: continuous
-    description:
-    code:
-    transform:
+- name: At1
+  type: continuous
+  description: First attribute.
+- name: At2
+  type: continuous
+  description: Second attribute.
```
Comment (trangdata) [inline datasets/banana/metadata.yaml]: Could we add 2 spaces in front of these lines, please?

File: datasets/banana/metadata.yaml [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+publication: 'Alcalá-Fdez J, Fernández A, Luengo J, Derrac J, García S, Sanchez L,
+  et al. KEEL Data-Mining Software Tool: Data Set Repository, Integration of Algorithms
+  and Experimental Analysis Framework. J Multiple Valued Log Soft Comput. 2011;17:
+  255–287.'
+task: classification
 keywords:
-  -
-  -
+- artificial
+- clustering
```
Comment (trangdata) [inline datasets/banana/metadata.yaml]: Similar here.

**Comment by trangdata** [comment]
Looks great @alexzwanenburg 💯 🌻  thank you so much for the contribution! I only have some minor comments!

**Comment by alexzwanenburg** [comment]
I fixed it. I actually use the yaml R-package to convert the metadata from a (nested) list to a yaml file, but this does not indent the nested bits correctly for some reason.

EDIT: I found the correct setting.

**Comment by alexzwanenburg** [comment]
I also updated the metadata file from the titanic dataset to use indenting for nested lists.

**Pull request merged**